src/arrays/Arrays.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace arrays;
+
+use arrays\ArraysInterface;
+
+class Arrays implements ArraysInterface
+{
+    /**
+     * @param array $input
+     * @return array
+     */
+    public function repeatArrayValues(array $input): array
+    {
+        $result = [];
+        $arrayCount = count($input);
+
+        for ($i = 0; $i < $arrayCount; $i++) {
+            for ($n = 0; $n < $input[$i]; $n++) {
+                $result[] = $input[$i];
+            }
+        }
+
+        return $result;
+    }
+
+    /**
+     * @param array $input
+     * @return int
+     */
+    public function getUniqueValue(array $input): int
+    {
+        $uniqueValue = 0;
+        $uniqueValues = [];
+
+        if (count($input) === 0) {
+            return $uniqueValue;
+        }
+
+        foreach ($input as $item) {
```

Comment:
could you please try to use php array functions?

Reply:
done